src/modules/[chain]/account/[address].vue

```diff
@@ -87,10 +109,305 @@ const totalValue = computed(() => {
   return format.formatNumber(value, '0,0.00');
 });
 
+// Calculate vesting percentage for vesting accounts
+const vestingPercentage = computed(() => {
```

Comment:
We're calculating vested percentage here no? That's [how we use it](https://github.com/fuel-infrastructure/explorer/blob/cdc7a8adcdeab433beb6b611cf75ea0fbf4faa30/src/modules/%5Bchain%5D/account/%5Baddress%5D.vue#L786)

```suggestion
// Calculate vested percentage for vesting accounts
const vestedPercentage = computed(() => {
```

Reply:
refactored in 5349dcc32cd806971e4246c4fc329f0a612cf835